### PR TITLE
Search function

### PR DESCRIPTION
**Pull request by KiveshVirasamy**
Please review my search function

### CODE REVIEW

**Review by MicNow27 — COMMENTED** [review]
In App.tsx, had to change `import "../dist/App.css";` to `import "./App.css";`

File: src/components/loadingScreen.tsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
-/**
- * Component for displaying a loading indicator.
- */
-function Loading(): JSX.Element {
+import React from "react";
+
+export function Loading(): JSX.Element {
```
Comment (MicNow27) [inline src/components/loadingScreen.tsx]: Is this used anywhere?

File: src/components/mangaInformation.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const GRADIENT_CLASSES =
+  "from-yellow-400 to-yellow-200 via-yellow-200 bg-gradient-to-t bg-clip-text text-transparent";
+const TAG_CLASSES =
+  "m-1 ml-0 rounded-full bg-yellow-600 w-fit px-2 text-[0.6rem] font-semibold uppercase";
+
+export function MangaInfoSheet({ mangaData, coverFile }: IMangaInfoProp) {
+  const [expandDescription, setExpandDescription] = useState(false);
+
+  const toggleDescription = () => setExpandDescription(!expandDescription);
```
Comment (MicNow27) [inline src/components/mangaInformation.tsx]: Not toggling for longer descriptions

File: src/components/searchManga.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { Field, Form, Formik } from "formik";
+import { IoMdSearch } from "react-icons/io";
+import { useNavigate } from "react-router-dom";
+
+export default function SearchManga() {
+  const navigate = useNavigate();
+
+  const handleSubmit = (values: { search: string }) => {
+    navigate(`./search/${values.search}`);
```
Comment (MicNow27) [inline src/components/searchManga.tsx:13]: Not navigating to a search result

File: src/components/mangaInformation.tsx (hunk 2 above)
Comment (KiveshVirasamy) [reply src/components/mangaInformation.tsx]: I made a change

File: src/components/searchManga.tsx (hunk 3 above)
Comment (KiveshVirasamy) [reply src/components/searchManga.tsx:13]: I made the change

File: src/components/loadingScreen.tsx (hunk 1 above)
Comment (KiveshVirasamy) [reply src/components/loadingScreen.tsx]: I removed this

**Review by MihlaliMgwebi — APPROVED** [review]
Looks Good. Approved.

**Pull request merged**